scripts/mainmenu_mapselector.js

```diff
@@ -134,6 +135,8 @@ class MapSelection {
 		// Set unchecked counter to -1, if you're using this you've got the hang of it, no more popups for you!
 		this.timesModeButtonsUnchecked = -1;
 
+		$.Msg(selectedMode);
```

Comment:
This was a mistake of mine that's now fixed on master, so you must be rebased wrong

Reply:
lol found the root cause. I was somehow carrying around your last commit from [refactor: Improve Gamemodes structure](https://github.com/momentum-mod/panorama/pull/40/commits/03db88f29a10540cd303b1fa86c376c3fddf8b7c). Fixed